integrations/datadog/main.py

```diff
@@ -28,6 +35,28 @@ def init_client() -> DatadogClient:
     )
 
 
+@ocean.on_resync(ObjectKind.TEAM)
+async def on_resync_teams(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    dd_client = init_client()
+
+    selector = cast(TeamResourceConfig, event.resource_config).selector
+
+    async for teams in dd_client.get_teams():
+        logger.info(f"Received teams batch with {len(teams)} teams")
+        if selector.include_members:
+            teams = await dd_client.enrich_teams_with_members(teams)
+        yield teams
```

Comment:
I think the `enrich_teams_with_members` should be just `get_team_members` and the logic of the merge will be in the main file

Reply:
Okay, makes sense. 👍 .